WinUIGallery/Navigation/NavigationRootPage.xaml.cs

```diff
@@ -201,7 +196,7 @@ private void AddNavigationMenuItems()
 
                 foreach (var item in group.Items)
                 {
-                    var itemInGroup = new Microsoft.UI.Xaml.Controls.NavigationViewItem() { IsEnabled = item.IncludedInBuild, Content = item.Title, Tag = item.UniqueId, DataContext = item };

```

Comment:
This change was intentional, see #1006. @niels9001, @karkarl thoughts?

Reply:
oh, I didn't know, should I return it?

Reply:
Let's wait for Niels or Karens answer on this but I would probably say yes.

Reply:
Yes, the icons in the submenu level were removed since they are not fluent icons, and the design team does not have capacity to create fluent icons for each of the controls. We will have to revert this change unfortunately.